primitive-types/impls/serde/src/serialize.rs

```diff
@@ -66,6 +66,7 @@ fn to_hex_raw<'a>(v: &'a mut [u8], bytes: &[u8], skip_leading_zero: bool) -> &'a
 #[derive(Debug, PartialEq, Eq)]
 pub enum FromHexError {
 	/// The `0x` prefix is missing.
+	#[deprecated(since = "0.3.2", note = "We support non 0x-prefixed hex strings")]
```

Comment:
one last nit: this will print deprecation warning when compiling this crate
could you put `#[allow(deprecated)]` above line 86?

Reply:
done